src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -174,4 +196,18 @@ private void setTaskAccessForNewReplicateLock(String chainTaskId, boolean isTask
         taskAccessForNewReplicateLock.replace(chainTaskId, isTaskBeingAccessedForNewReplicate);
     }
 
+    public boolean isConsensusReached(Task task) {
```

Comment:
```suggestion
    public boolean isConsensusReached(String chainTaskId) {
```
What do you think ?

Reply:
Well, yes, that's probably better. I'll update that!